line.go

```diff
@@ -35,7 +43,7 @@ func (l Line) ContainsPoint(pt [2]float64) bool {
 	return minx <= pt[0] && pt[0] <= maxx && miny <= pt[1] && pt[1] <= maxy
 }
 
-// ContainsPoint checks to see if the given pont lines on the linesegment. (Incliding the end points.)
+// ContainsPointBigFloat checks to see if the given pont lines on the linesegment. (Incliding the end points.)
 func (l Line) ContainsPointBigFloat(pt [2]*big.Float) bool {
```

Comment:
Let's fix the spelling error here. (Include, point, and lies  are spelled wrong.)
```
// ContainsPointBigFloat checks to see if the given point lies on the line segment. (Including the endpoints.)
```